README.rst

```diff
@@ -50,6 +44,31 @@ b) from data:
   >>> img.save(bio, format='jpeg')
   >>> pc.uploadfile(data=bio.getvalue(), filename="image.jpg", path='/path-to-pcloud-dir')
 
+Usage of PyFilesystem with opener
+
+ >>> from fs import opener
+ >>> opener.open_fs('pcloud://email%40example.com:SecretPassword@/')
+ <pCloudFS>
```

Comment:
Can you please use the same indention as below.

Reply:
Sorry for the delay, I must have missed this.
What do you mean exactly?
`opener.open_fs` doesn't need any indentention like with the first `with opener.open_fs(` below.